Core/include/Acts/EventData/TrackStateProxy.hpp

```diff
@@ -667,9 +676,41 @@ class TrackStateProxy {
   /// @note This is mainly to copy explicitly a projector from one state
   ///       to another. If you have a projection matrix, set it with
   ///       `setProjector`.
+  /// @deprecated Use setProjector(span) instead
+  //[[deprecated("use setProjector(span) instead")]]
   void setProjectorBitset(ProjectorBitset proj) requires(!ReadOnly) {
+    BoundMatrix projMatrix = bitsetToMatrix<BoundMatrix>(proj);
+    FullProjectorMapping fullMapping =
+        projectorToIndices<eBoundSize>(projMatrix);
+    setFullProjectorMapping(fullMapping);
+  }
+
+  FullProjectorMapping fullProjectorMapping() const {
```

Comment:
Just to confirm my understanding: *full* here means it's bound-sized, and the opposite is measurement sized, so dynamic and equal to `calibratedSize`?